tanner/tests/test_api_server.py

```diff
@@ -10,8 +11,8 @@ class TestAPIServer(AioHTTPTestCase):
     def setUp(self):
         self.serv = server.ApiServer()
 
-        postgres = mock.Mock()
-        postgres.close = mock.Mock()
+        postgres = AsyncMock()
+        postgres.close = AsyncMock()
```

Comment:
close is not a coroutine https://aiopg.readthedocs.io/en/stable/core.html#aiopg.Pool.close